src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 1.4;
+  color: $primary-color;
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
it's better to set for each element on the page (use `*` selector)
Note that `box-sizing` is the only property allowed to be set in `*` selector (exception)